docs/zh/docs/deployment.md

```diff
@@ -0,0 +1,390 @@
+# 部署
+
+部署 **FastAPI** 应用相对比较简单。
+
+根据特定使用情况和使用工具有不同的部署方式。
+
+接下来的章节，你将了解到一些关于部署方式的内容。
+
+## FastAPI 版本
+
+许多应用和系统已经在生产环境使用 **FastAPI**。其测试覆盖率保持在 100%。但该项目仍在快速开发。
+
+我们会经常加入新的功能，定期错误修复，同时也在不断的优化项目代码。
+
+这也是为什么当前版本仍然是 `0.x.x`，我们以此表明每个版本都可能有重大改变。
+
+现在就可以使用 **FastAPI** 创建生产应用（你可能已经使用一段时间了）。你只需要确保使用的版本和代码其他部分能够正常兼容。
+
+### 指定你的 `FastAPI` 版本
+
+首先，要指定的 **FastAPI** 版本，选择一个能够正确运行你的应用的最新版本。
+
+例如，应用中正在使用版本 `0.45.0`。
+
+如果你使用 `requirements.txt` 文件，你可以这样指定版本：
+
+```txt
+fastapi==0.45.0
+```
+
+这表明您将使用 `0.45.0` 版本的 `FastAPI`。
+
+或者您也可以这样指定：
+
+```txt
+fastapi>=0.45.0,<0.46.0
+```
+
+这表明你将使用 `0.45.0` 及以上，但低于 `0.46.0` 的版本，例如，`0.45.2` 依然可以接受。
+
+如果使用其他工具管理你的安装，比如 Poetry，Pipenv，或者其他工具，它们都有各自指定包的版本的方式。
+
+### 可用版本
+
+你可以在 [发行说明](release-notes.md){.internal-link target=_blank} 中查看可用的版本（比如：检查最新版本是什么）。
+
+
+### 关于版本
+
+FastAPI 遵循语义版本控制约定，`1.0.0` 以下的任何版本都可能加入重大变更。
```

Comment:
FastAPI 遵循语义化版本控制规范，`1.0.0` 以下的任何版本都可能加入不兼容的变更。

Reply:
Definition of `Convension`  from Oxford:
1. a way in which something is usually done. 
2. an agreement between states covering particular matters, especially one less formal than a treaty.

"规范" means "Standard, specification or regulation", that's not the right word for it.
So the meaning of `Convension` should be "约定，协定，习俗"  rather than "规范"

Reply:
Definition of Convension from Oxford:

a way in which something is usually done.
an agreement between states covering particular matters, especially one less formal than a treaty.
"规范" means "Standard, specification or regulation", that's not the right word for it.
So the meaning of Convension should be "约定，协定，习俗" rather than "规范"

Reply:
I think you are right.